### PR TITLE
chore(token): change get to post

### PR DESCRIPTION
**Pull request by IsaacGzOv**
changed route /users/new-password from get to post

### CODE REVIEW

File: src/controllers/token-controller.js [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
-  }
-  try {
-    await tokenService.sendTokentoMail(mail);
-    res.status(202).send({ status: 'OK', message: 'Mail send' });
-  } catch (error) {
-    res.status(404).send({
+  const result = token.validate(req.body);
+  console.log(req.body.mail);
+  if (result.error) {
+    res.status(400).send({
```
Comment (gonzajo) [inline src/controllers/token-controller.js]:
```suggestion
  const { error } = token.validate(req.body);
  if (error) {
    res.status(400).send({
```

File: src/controllers/token-controller.js [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
       status: 'FAILED',
-      data: { error: error?.message || error },
+      data: { error: result.error.details },
```
Comment (gonzajo) [inline src/controllers/token-controller.js]:
```suggestion
      data: { error: error.details },
```

**Review by gonzajo — CHANGES_REQUESTED** [review]

File: src/controllers/token-controller.js [hunk 3]
```diff
@@ -1,23 +1,28 @@
+const joi = require('joi');
 const tokenService = require('../services/token-service');
 
+const token = joi.object({
+  mail: joi.string().email().max(100).required(),
+});
```
Comment (gonzajo) [inline src/controllers/token-controller.js:6]: why a token it has been validate with email?

File: src/controllers/token-controller.js [hunk 4]
```diff
@@ -1,23 +1,28 @@
...
-      data: { error: "Parameter ':mail' can not be empty" },
-    });
-  }
-  try {
-    await tokenService.sendTokentoMail(mail);
-    res.status(202).send({ status: 'OK', message: 'Mail send' });
-  } catch (error) {
-    res.status(404).send({
+  const { error } = token.validate(req.body);
+  console.log(req.body.mail);
```
Comment (gonzajo) [inline src/controllers/token-controller.js]: remove this

**Review by gonzajo — CHANGES_REQUESTED** [review]

File: src/controllers/token-controller.js (hunk 3 above)
Comment (IsaacGzOv) [reply src/controllers/token-controller.js:6]: i tink i need to change de name of the var, in the request i only ask for de mail, so thts why is validate as mail

**Review by gonzajo — APPROVED** [review]

**Pull request merged**